Applications/Telemetry/telemetry.py

```diff
@@ -181,22 +218,20 @@ def rawTelemetry():
     provides non-SQLite database dequeu results. Each query can pop data off of
     a queu, thus ensuring there are no duplicates and acting like a proxy with
     decoded data instead of BASE64 encoded data.
+
+    :return: JSON formatted string with data or error. HTTP204 if no data
     """
+    #  Initialize local variables
+    stationData = []
```

Comment:
Is needed due to potential for no stations to be present prior to len() check later when no variable is present.